tests/integration/llm/client.py

```diff
@@ -772,6 +772,14 @@ def get_model_name():
     }
 }
 
+handler_performance_model_spec = {
+    "tiny-llama-model": {
+        "batch_size": [1, 128],
```

Comment:
In our performance tests previously, we have observed a pretty noticeable change in behavior around batch size 64. Could we include some more batch-sizes here? Maybe [1, 8, 32, 64, 128, 256]?

Reply:
We can if that is preferable, the original goal was to get a no batch and a high batch baseline set. And I see what you are looking to accomplish with this, making this more diagnostic and flexible. Current runtime for the test is about 34 minutes, so I think I will make some changes to support the expansion but keep it to these two values until we discuss in ops next week.

Reply:
yea, we can easily change it in the future so i'm cool with that.